### PR TITLE
Test branch

### PR DESCRIPTION
**Pull request by DylanTinianov**

### CODE REVIEW

File: .idea/workspace.xml [hunk 1]
```diff
@@ -0,0 +1,697 @@
...
+    <entry file="file://$PROJECT_DIR$/Generate_Response/Check_POSTags.py">
+      <provider selected="true" editor-type-id="text-editor">
+        <state relative-caret-position="75">
+          <caret line="5" column="0" selection-start-line="5" selection-start-column="0" selection-end-line="5" selection-end-column="0" />
+          <folding />
+        </state>
+      </provider>
+    </entry>
+  </component>
+</project>
```
Comment (lpan) [inline .idea/workspace.xml]: Usually we dont commit editor cache to git :) you can add

```
.idea
```

to our .gitignore file

File: Generate_Response/Check_POSTags.py [hunk 2]
```diff
@@ -0,0 +1,33 @@
+""" Use POS tags to find useful sentence parts"""
```
Comment (lpan) [inline Generate_Response/Check_POSTags.py]: Correct me if I am wrong I think python module name should follows lowercase snake case convention (just like function names!).

So in this case, it should be
`generate_response/check_pos_tags.py`

File: SexxiBot.py [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+import Behaviours
+import random
+
+__author__ = "Waterloo SE'XXI"
+
+keywords = Behaviours.KeyWords()
+responses = Behaviours.Responses()
+
+
+class SexxiBot:
```
Comment (lpan) [inline SexxiBot.py]: ayyyy

File: Generate_Response/Format_Response.py [hunk 4]
```diff
@@ -0,0 +1,45 @@
+# Is used when a key response is not triggered.
+import random
+
+
+def craft_response(noun, pronoun, verb, adj, prep, text_len):
+    """ Creates a response based on the user's input """
+    print noun, pronoun, verb, adj, prep, text_len
```
Comment (lpan) [inline Generate_Response/Format_Response.py]: is it python2 :open_mouth: ?

Comment (DylanTinianov) [inline Generate_Response/Format_Response.py]: Ya 😮

Comment (lpan) [inline Generate_Response/Format_Response.py]: :+1:

**Pull request closed without merging**